packages/remirror__extension-file/src/file-extension.tsx

```diff
@@ -219,6 +222,34 @@ export class FileExtension extends NodeExtension<FileOptions> {
     };
   }
 
+  @keyBinding({ shortcut: ['Backspace', 'Delete'] })
+  backspaceKey(props: KeyBindingProps): boolean {
```

Comment:
<code>backspace<strong><em>Key</strong></em></code> => <code>backspace<strong><em>Shortcut</strong></em></code> seems to be more fitting with the existing conventions.